micromamba/src/env.cpp

```diff
@@ -163,4 +165,37 @@ set_env_command(CLI::App* com)
             }
             t.print(std::cout);
         });
+
+    auto* remove_subcom = com->add_subcommand("remove", "Remove an environment");
+    init_general_options(remove_subcom);
+    init_prefix_options(remove_subcom);
+
+    remove_subcom->callback(
+        []()
+        {
+            // Remove specs if exist
+            remove(MAMBA_REMOVE_ALL);
+
+            auto& ctx = Context::instance();
+            if (!ctx.dry_run)
+            {
+                auto& prefix = ctx.target_prefix;
+                // Remove env directory and all its content
+                fs::remove_all(env::expand_user(prefix));
```

Comment:
So I just noticed that we might be forgetting one annoying case here. On Windows, if a file is in use, we cannot remove it (only rename) :/

conda / mamba solves this by renaming the files to `<filename>.mamba_trash`